src/cloud-api-adaptor/install/overlays/docker/kustomization.yaml

```diff
@@ -7,7 +7,7 @@ bases:
 images:
   - name: cloud-api-adaptor
     newName: quay.io/confidential-containers/cloud-api-adaptor # change image if needed
-    newTag: latest
+    newTag: a7dc1190e98c1d2377b9d235f1e3d91736ce7d60
```

Comment:
Hi @stevenhorsman !

docker needs the dev image if I'm not mistaken.

Reply:
You are correct. Good catch!